### ISSUE TITLE
Enhancement suggestion : listview instead of layout_height wrap_content, use XXXdp

### ISSUE THREAD

**Issue by x4080** [Mauker1/MaterialSearchView#28]
This will allow scroll when keyboard is shown, especially in the landscape mode

And speaking of landscape, maybe if 2 layout available for XXXdp sizes for portrait and landscape it will be better

Thanks

**Comment by Mauker1** [comment]
Could you explain your suggestion a little better? Maybe provide an example.

**Comment by x4080** [comment]
I mean using static height so that the list is above the keyboard and can be scrolled

**Comment by Mauker1** [comment]
I think that will be hard to get right, given the number of screen sizes, soft keyboard sizes (and even 3rd party keyboards). And if the user dismisses the keyboard, the list won't get any bigger.

**Comment by x4080** [comment]
that's correct, i just use some value for portrait and smaller value for landscape, Google search also use static size btw

**Comment by Mauker1** [comment]
@AdamMc331 Any thoughts on this one?
